apps/docs/.vitepress/plugins/utils/extractImports.ts

```diff
@@ -2,6 +2,13 @@ import { FuelError, ErrorCode } from '@fuel-ts/errors';
 import fs from 'fs';
 import { IMPORT_REGEXP, IMPORT_START_REGEXP } from '../snippetPlugin';
 
+/**
+ * @hidden
+ * 
+ * Constant used to prefix type imports.
+ */
```

Comment:
```suggestion
/**
 * Constant used to prefix type imports.
 */
```

Hidden is redundant here. We don't build tsdoc docs here.